docs/pages/distribution/app-transfers.md

```diff
@@ -1,13 +1,16 @@
 ---
 title: App Transfers
+hideTOC: true
 ---
 
-When you upload an iOS app to App Store Connect and distribute it through the App Store, it becomes (semi-)permanently associated with your Apple Developer account. This means that any future updates to your app must go through your Apple account. Apple provides a process called [App Transfer](https://help.apple.com/app-store-connect/#/deved688524f) as a way around this, for cases in which you want to transfer the ownership and maintainability of one of your apps to a different Apple Developer account.
+import { BoxLink } from '~/ui/components/BoxLink';
 
-One of the [criteria for an app to be eligible to transfer](https://help.apple.com/app-store-connect/#/devaf27784ff) is that `"No version of the app can use a Passbook entitlement."`. Until December 11, 2018, apps built with Expo SDK 30 or 31 included the `Payments` module, which necessitates linking a file called `PassKit.framework`. This file includes the native APIs for interacting with the native iOS payments functionality. All SDK 30 and 31 apps built before this date included this file -- this is because we always include the native code for all modules in every build, even modules that you don't import in your JS. (Among other reasons, this makes it possible for you to use different modules in updates without having to build a new binary.)
+There are two different representations of your app to consider when handing over ownership to another entity: the app as it exists on Expo Application Services (in order to create builds with EAS Build, send updates with EAS Update, etc.) and the app records on the Apple App Store / Google Play Store (in order to distribute the app to end-users). The following guides explain how to handle app transfers in each of these cases.
```

Comment:
```suggestion
There are two different representations of your app to consider when handing over ownership to another entity: the app as it exists on Expo Application Services (to create builds with EAS Build, send updates with EAS Update, etc.) and the app records on the Apple App Store/Google Play Store (to distribute the app to end-users). The following guides explain how to handle app transfers in each case.
```

As per our style guide, let's [avoid adding spaces before and after `/`](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#do-not-add-a-space-between-the-words-and-).